react/features/e2ee/components/E2EELabel.tsx

```diff
@@ -24,18 +24,25 @@ export interface IProps extends WithTranslation {
 
 
 const E2EELabel = ({ _e2eeLabels, _showLabel, t }: IProps) => {
-    if (!_showLabel) {
-        return null;
+    const [olmAvailable, setOlmAvailable] = useState(Boolean(window.Olm)); 
+
+    useEffect(() => {
+        const checkOlm = () => setOlmAvailable(Boolean(window.Olm)); 
```

Comment:
I don't think supporting someone doing `delete window.Olm` is something we should guard against.

If someone is being that dumb they can deal with the consequences themselves.